src/App/Fig.purs

```diff
@@ -65,42 +65,40 @@ setInputView x δvw fig = fig
    { in_views = insert x (lookup x fig.in_views # join <#> δvw) fig.in_views
    }
 
-combineVals :: forall g. Graph g => Set (DVertex' (Val Vertex)) -> Set (DVertex' (Val Vertex)) -> Set DVertex -> g -> g -> Array (Val (SelState 𝕊))
-combineVals persistents transients inerts gPersistent gTransient =
-   vals𝕊
+combineVals :: forall g. Graph g => Set (DVertex' (Val Vertex)) -> Set (DVertex' (Val Vertex)) -> Set DVertex -> SelTracker g -> Array (Val (SelState 𝕊))
+combineVals persistents transients inerts g =
```

Comment:
I think we said we would treat these two argument using the same `Selection` pattern